src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int bottomBase;
+    private int topBase;
+    private int hight;

```

Comment:
There is a typo in the field name `hight`. It should be corrected to `height` to maintain clarity and correctness.